utils.py

```diff
@@ -235,4 +239,97 @@ def check_and_fill(args, target_len):
 
 
 
+## Augmentation
+class SegmentationAlbumentationsTransform(ItemTransform):
```

Comment:
Is there a reason why you used a class? The class seems to make sense, however, I am wondering if we are making the UNet Framework more confusing or fragile if we are now also starting to implement classes, as most functionality is implemented in functions so far.